hack/generate.sh

```diff
@@ -56,6 +56,16 @@ jsonnet -c -J vendor -m monitoring-satellite/manifests \
         slackWebhookURLInfo: 'http://fake.url.info',
         slackChannelPrefix: '#fake_channel',
         pagerdutyRoutingKey: 'fakeR0uT1GNKEY',
+
+        IDE: {
+            pagerdutyRoutingKey: 'fake-IDE-pd-key',
+        },
+        webapp: {
+            pagerdutyRoutingKey: 'fake-webapp-pd-key',
+        },
+        platform: {
+            slackWebhookURLCritical: 'https://platfrom.slack.webhook.com',
+        },
```

Comment:
This is an example configuration that we'll have to add ArgoCD apps to enable team-specific alerting routes. 

They can choose between slack or pagerduty, and the company-wide routes will be used if a team doesn't have such configuration.